libcrux-ml-kem/src/kem/kyber/helper.rs

```diff
@@ -0,0 +1,57 @@
+/// The following macros are defined so that the extraction from Rust to C code
+/// can go through.
+
+#[cfg(not(hax))]
+#[macro_export]
+macro_rules! cloop {
```

Comment:
Let's at least hide the docs for these two.

```suggestion
#[doc(hidden)]
```